--- wine/programs/wineboot/wineboot.c	2025-07-30 22:02:41.000000000 +0800
+++ wine-b/programs/wineboot/wineboot.c	2026-03-16 12:00:00.000000000 +0800
@@ -535,7 +535,9 @@
     SYSTEM_BASIC_INFORMATION *data;
 
     data = heap_alloc( sizeof(*data) );
     if (data) memset( data, 0, sizeof(*data) );
+#if defined(__i386__) || defined(__x86_64__)
     initialize_xstate_features( data );
+#endif
 
     RtlGetNativeSystemInformation( SystemBasicInformation, data, sizeof(*data), NULL );